cookbook/evaluation_get_started.ipynb

```diff
@@ -109,15 +109,17 @@
     },
     {
       "cell_type": "code",
-      "execution_count": 4,
+      "execution_count": null,
       "metadata": {
         "id": "8viV4KT5RMjA"
       },
       "outputs": [],
       "source": [
         "from langfuse import Langfuse\n",
         "\n",
-        "langfuse = Langfuse()"
+        "langfuse = Langfuse()\n",
+        "\n",
+        "langfuse.auth_check()"
```

Comment:
needs comment why this is optional, otherwise this will be used in prod (unnecessary)